core/modules/node/node.module

```diff
@@ -1826,6 +1843,24 @@ function node_menu() {
   return $items;
 }
 
+/**
+ * Access callback: Checks a user's permission for previewing a node.
+ *
+ * @param EntityInterface $node
+ *   The node entity to be previewed.
+ *
+ * @return bool
+ *   TRUE if the user has access to the preview, FALSE otherwise.
+ *
+ * @see node_menu()
+ */
+function node_access_preview(EntityInterface $node) {
```

Comment:
As this is made specifically for nodes and not other entities, I'd use `Node` as the typehint.

Reply:
done